apps/dsb-client-gateway-api/src/app/modules/message/dto/response/get-message-response.dto.ts

```diff
@@ -98,11 +99,19 @@ export class GetMessagesResponseDto {
   })
   transactionId: string;
 
-  @IsBoolean()
+  @IsString()
+  @IsEnum(EncryptionStatus)
   @ApiProperty({
     description: 'transactionId sent to message for idempotency',
```

Comment:
This description should be updated

Reply:
@nik-ewf just fixed it.
this is also happening in `schemaType`